src/ray/gcs/tables.cc

```diff
@@ -475,10 +475,14 @@ std::string ProfileTable::DebugString() const {
   return Log<UniqueID, ProfileTableData>::DebugString();
 }
 
-Status JobTable::AppendJobData(const JobID &job_id, bool is_dead) {
+Status JobTable::AppendJobData(const JobID &job_id, bool is_dead, int64_t timestamp,
```

Comment:
Could just compute the `timestamp` inside this method instead of passing it in, but I don't really have a preference.

Reply:
I prefer to have it outside, that makes it possible for somebody to use this function to use this function with a different timestamp than the current time.